cypress/integration/tx_modal.spec.js

```diff
@@ -0,0 +1,203 @@
+const TEST_SAFE = 'rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const RECIPIENT_ENS = 'diogo.eth'
+const RECIPIENT_ADDRESS = '0x6a5602335a878ADDCa4BF63a050E34946B56B5bC'
+const SAFE_NONCE = '6'
+
+describe('Tx Modal', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    // Open the Safe used for testing
+    cy.visit(`/${TEST_SAFE}`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  describe('Send funds modal', () => {
+    describe('Send Funds form', () => {
+      before(() => {
+        // Open Send Funds Modal
+        cy.contains('New Transaction').click()
+        cy.contains('Send funds').click()
+      })
+
+      it('should display Send Funds modal with all the form elements', () => {
+        // Modal header
+        cy.get('form')
+          .prev()
+          .prev()
```

Comment:
This is testing an implementation detail - that it has to be exactly the second previous sibling of the form element. Why not check for `cy.contains('Send funds')` directly instead of moving from the `form` element as a reference point?